configure.ac

```diff
@@ -2309,6 +2318,7 @@ fi
 echo "  linker flags                : $LDFLAGS $LIBS"
 echo "  prefix                      : $prefix"
 echo "  linking mode                : $linking_mode"
+echo "  classic linking mode        : ${enable_force_classic_linking:=no}"
```

Comment:
As this is an XCode (clang)-only flag, can we skip it from this summary to avoid confusion?
Alternatively, we could give it a better name that suggests XCode, but I don't think it's worth adding such a specific thing to the summary.

Reply:
My bet is that this is a clang issue, so it might have an effect on other OS and clang versions later on too.

Yepp, I can hide it if you wish of course.

Reply:
I couldn't find such clang flag outside of Xcode